templates/default/070-components/UI-framework/Input/_ui-component_rating.scss

```diff
@@ -0,0 +1,95 @@
+@use "../../../010-settings/" as *;
+
+.il-input-rating .il-input-rating__text {
```

Comment:
Maybe some nesting could improve the overall structure and readability and also give us a chance to really use the BEMIT naming convention:
```SCSS
.c-input-rating {
    &__text {
    }
    &__stars {
    }
    &__star {
        &--empty {
        }
    }
}
```